client/src/client.rs

```diff
@@ -957,6 +959,10 @@ pub trait RpcApi: Sized {
             "rescanblockchain", handle_defaults(&mut args, &[0.into(), null()]))?;
         Ok((res.start_height, res.stop_height))
     }
+
+    fn scan_txout_set(&self, descriptors: &[ScanTxoutRequest]) -> Result<json::ScanUtxoResult> {
+        self.call("scantxoutset", &["start".into(), into_json(descriptors)?])
```

Comment:
What about the `abort` and `status` actions?

If this is only meant to support starting a scan, maybe rename this to something like `scan_txout_set_start()`?

Reply:
I'm super confused by these actions. We model `start` as a blocking action, so why enable abort? If you see a use case I'll add two functions for these, but I didn't see the use case.

Reply:
I'm not sure really, but if its supported in Core then there probably is some merit to it and I think that bitcoincore-rpc should ideally support it too.

But at the very least, if the RpcApi method provides a subset of the functionality of the RPC command, it should not use the same name. I think its alright to add support just for `start` for now, if we just give the method a name that hinted at that.

Reply:
I guess I could just add `_blocking` to the name to indicate it doesn't support the (probably?) somewhat async API.

Reply:
fixed by renaming

Reply:
The `_blocking` is a bit odd because all methods are blocking, this might create the impression that non-`_blocking` methods are not blocking.